src/docs/faqs/faqs.md

```diff
@@ -1,48 +1,55 @@
 ---
 layout: docs
 title: Frequently Asked Questions
-published: 2022-12-14
+published: 2023-05-11
 add toc: true 
 add support: true 
 add sidebar: _sidebars/mainSidebar.md
 article_status: published
-todo: 
-1. Search for existing Questions (CMR)
-2. Provide answers (CMR)
-3. Include links to corresponding KB article (CMR)
-4. Open FAQ-section to public (CMR)
 ---
 
 ## General
 
-**Q:**  
-
-**A:**  
+<details><summary><span style="color: #1fc2a7;font-size:1.2em">
+Which format must be used to enter a date?
+</summary>
+String formatted as ISO8601 date: YYYY-MM-DD
+</details>
+
+<details><summary><span style="color: #1fc2a7;font-size:1.2em">
+Is it necessary to fill in the empty fields in the investigation-file manually? 
+</summary>
+No! Although it is possible to fill in the workbook manually, we recommend using <a href="https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/implementation/ArcCommander.html">ARC Commander</a> to add this metadata.
```

Comment:
I would avoid exclamation marks, could be interpreted as us yelling at them or that the answer is so obvious. 
And maybe unify all answers to "No, ...", "Yes, ...", "It depends ...".

Reply:
Was applied everywhere where suitable.